eZ/Bundle/EzPublishCoreBundle/Imagine/VariationPathGenerator.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
```

Comment:
[Like this?](https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/Core/Search/Common/FieldNameGenerator.php#L2-L8)